ops/terraform/services/migrator/deploy.groovy

```diff
@@ -90,10 +105,13 @@ String monitorMigrator(Map args = [:]) {
         // 2. if the message body contains a non "0/0" (running) value, return it
         for (msg in messages) {
             migratorStatus = msg.body.status
+            println "Migrator schema version is at ${msg.body.schema_version}"
+            schemaVersion = msg.body.schema_version
```

Comment:
swap the order of these two lines and use ${schemaVersion} in your `println`

Reply:
done